Assets/Tests/InputSystem/CoreTests_HighLevelAPI.cs

```diff
@@ -0,0 +1,753 @@
+#if UNITY_INPUT_SYSTEM_ENABLE_GLOBAL_ACTIONS_API
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.Controls;
+using UnityEngine.InputSystem.HID;
+using UnityEngine.InputSystem.Layouts;
+using UnityEngine.InputSystem.LowLevel;
+using UnityEngine.InputSystem.Utilities;
+using UnityEngine.LowLevel;
+using UnityEngine.PlayerLoop;
+using UnityEngine.TestTools;
+using UnityEngine.TestTools.Constraints;
+using UnityEngine.TestTools.Utils;
+using GamepadButton = UnityEngine.InputSystem.LowLevel.GamepadButton;
+using Input = UnityEngine.InputSystem.Input;
+using Is = NUnit.Framework.Is;
+using Random = UnityEngine.Random;
+
+internal partial class CoreTests
+{
+    // TODO rename HighLevelAPI to something that makes sense
+    // one perspective is to see this API as an evolution of input manager API, aka input manager 2.0
+    [Test]
+    [Category("HighLevelAPI")]
+    public void HighLevelAPI_CanQueryControl()
```

Comment:
This whole test can be deleted if we remove Input.IsPressed/WasPressedThisFrame/WasReleasedThisFrame